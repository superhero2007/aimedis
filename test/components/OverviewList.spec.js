import React from 'react';
import { mount } from 'enzyme';

import OverviewList from 'components/OverviewList';

function setup() {
  return mount(
    <OverviewList
      list={[
        {
          id: 1,
          right: 'IMac.png',
          src: 'mobile.png',
          subtitle: 'In every actual browser',
          title: 'Fully responsive',
        }]}
    />);
}

describe('OverviewList', () => {
  const wrapper = setup();

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
