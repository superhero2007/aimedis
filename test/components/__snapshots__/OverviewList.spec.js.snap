// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OverviewList should render properly 1`] = `"<div class=\\"app__overview__list\\"><div class=\\"app__overview__list__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-5 left\\"><div class=\\"app__overview__list__content__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__overview active\\"><div class=\\"app__overview__internal\\"><div class=\\"app__overview__internal__content\\"><div class=\\"app__overview__internal__content__order\\"><p class=\\"app__overview__internal__content__order__text body\\">1</p></div><div class=\\"app__overview__internal__content__paragraph\\"><p class=\\"app__overview__internal__content__paragraph__title title\\">Fully responsive</p><p class=\\"app__overview__internal__content__paragraph__subtitle body\\">In every actual browser</p></div><div class=\\"app__overview__internal__content__next ml-auto\\"><span class=\\"i-long-arrow-right d-none d-lg-inline\\"></span><span class=\\"i-long-arrow-down d-inline d-lg-none\\"></span></div></div></div><div class=\\"app__overview__external\\"><div class=\\"app__overview__external__image\\"><img src=\\"\\" alt=\\"No Asset\\"></div></div></div></div></div></div><div class=\\"col-lg-7 right d-none d-lg-block app__overview__list__content__mobile__image\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><img src=\\"\\" alt=\\"No Asset\\"><div class=\\"mask\\"><img src=\\"\\" alt=\\"No Asset\\"></div></div></div></div></div></div>"`;

exports[`OverviewList should render properly 2`] = `"<div class=\\"app__overview__list\\"><div class=\\"app__overview__list__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-5 left\\"><div class=\\"app__overview__list__content__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__overview active\\"><div class=\\"app__overview__internal\\"><div class=\\"app__overview__internal__content\\"><div class=\\"app__overview__internal__content__order\\"><p class=\\"app__overview__internal__content__order__text body\\">1</p></div><div class=\\"app__overview__internal__content__paragraph\\"><p class=\\"app__overview__internal__content__paragraph__title title\\">Fully responsive</p><p class=\\"app__overview__internal__content__paragraph__subtitle body\\">In every actual browser</p></div><div class=\\"app__overview__internal__content__next ml-auto\\"><span class=\\"i-long-arrow-right d-none d-lg-inline\\"></span><span class=\\"i-long-arrow-down d-inline d-lg-none\\"></span></div></div></div><div class=\\"app__overview__external\\"><div class=\\"app__overview__external__image\\"><img src=\\"\\" alt=\\"No Asset\\"></div></div></div></div></div></div><div class=\\"col-lg-7 right d-none d-lg-block app__overview__list__content__mobile__image\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><img src=\\"\\" alt=\\"No Asset\\"><div class=\\"mask\\"><img src=\\"\\" alt=\\"No Asset\\"></div></div></div></div></div></div>"`;
